data_generation/CRF2013.py

```diff
@@ -0,0 +1,1631 @@
+    cats["2.B.9.a"]["children"] = [["2.B.9.a.i", "2.B.9.a.ii"]]
+    ncats["2.B.9.a.i"] = {"title": "Production of HCFC-22"}
+    ncats["2.B.9.a.ii"] = {"title": "Other (Please Specify"}
+    cats["2.B.9.b"]["children"] = [["2.B.9.b.i", "2.B.9.b.ii", "2.B.9.b.iii"]]
+    ncats["2.B.9.b.i"] = {"title": "Production of HFC-134a"}
+    ncats["2.B.9.b.ii"] = {"title": "Production of SF6"}
+    ncats["2.B.9.b.iii"] = {"title": "Other (Please Specify)"}
+    # Metal industry
+    cats["2.C.3"]["children"] = [[f"2.C.3.{x}" for x in "ab"]]
+    ncats["2.C.3.a"] = {"title": "By-Product Emissions"}
+    ncats["2.C.3.b"] = {"title": "F-Gases Used in Foundries"}
+    # Product uses as substitutes for ODS
+    cats["2.F.1"]["children"] = [[f"2.F.1.{x}" for x in "abcdef"]]
+    del cats["2.F.1.a"]
+    del cats["2.F.1.b"]
+    ncats["2.F.1.a"] = {"title": "Commercial Refrigeration"}
+    ncats["2.F.1.b"] = {"title": "Domestic Refrigeration"}
+    ncats["2.F.1.c"] = {"title": "Industrial Refrigeration"}
+    ncats["2.F.1.d"] = {"title": "Transport Refrigeration"}
+    ncats["2.F.1.e"] = {"title": "Mobile Air-Conditioning"}
+    ncats["2.F.1.f"] = {"title": "Stationary Air-Conditioning"}
+    cats["2.F.4"]["children"] = [["2.F.4.a", "2.F.4.b"]]
+    ncats["2.F.4.a"] = {"title": "Metered dose Inhalers"}
+    ncats["2.F.4.b"] = {"title": "Others (Please Specify)"}
+    cats["2.F.6"]["children"] = [["2.F.6.a", "2.F.6.b"]]
+    ncats["2.F.6.a"] = {"title": "Emissive"}
+    ncats["2.F.6.b"] = {"title": "Contained"}
+    # Other product manufacture and use
+    del cats["2.G.1"]["children"]
+    for x in "abc":
+        del cats[f"2.G.1.{x}"]
+    cats["2.G.2"]["children"] = [[f"2.G.2.{x}" for x in "abcde"]]
+    del cats["2.G.2.c"]
+    ncats["2.G.2.c"] = {"title": "Soundproof Windows"}
+    ncats["2.G.2.d"] = {"title": "Adiabatic Properties: Shoes and Tyres"}
+    ncats["2.G.2.e"] = {"title": "Other (Please Specify)"}
+
+    # Agriculture and LULUCF are separated in CRF but one category in IPCC2006
+    # Thus we have to build the complete tree and delete the IPCC categories
+    # first remove IPCC2006 category 3 (AFOLU)
+    cats_to_remove = [cat for cat in cats.keys() if cat[0] == "3"]
+    for cat in cats_to_remove:
+        del cats[cat]
+
+    # Table3s1/2 - Summary tables, only used for top level category
+    ncats["3"] = {
+        "title": "Total Agriculture",
+        "children": [
+            [f"3.{x}" for x in "ABCDEFGHIJ"],
+            ["M.3.LV"] + [f"3.{x}" for x in "CDEFGHIJ"],
+        ],
+    }
+    ncats["M.3.LV"] = {
+        "title": "Livestock",
+        "children": [["3.A", "3.B"]],
+    }
+
+    # Table3.A - Enteric fermentation
+    ncats["3.A"] = {
+        "title": "Enteric Fermentation",
+        "children": [["3.A.1", "3.A.2", "3.A.3", "3.A.4"]],
+    }
+    ncats["3.A.1"] = {
+        "title": "Cattle",
+        "children": [["3.A.1.Aa", "3.A.1.Ab"], ["3.A.1.Ba", "3.A.1.Bb", "3.A.1.Bc"]],
```

Comment:
```suggestion
        "children": [["3.A.1.Aa", "3.A.1.Ab"], ["3.A.1.Ba", "3.A.1.Bb", "3.A.1.Bc"], ["3.A.1.C"]],
```

I think 3.A.1.C was missing from the children here.